### DISCUSSION TITLE
How can I flash the Slint UI to an ESP development board based on Rust?

### DISCUSSION THREAD

**Discussion by hhhdsdlfsdh** [slint-ui/slint#4222] [Q&A]
The UI flash error for slint is shown below
(https://github.com/slint-ui/slint/assets/109570454/0d076571-ffe7-4bc9-ab4c-99ca1dc3e26a)

**Comment by ogoffart** [comment]
(Converted the issue into a discussion, to keep the bug tracker for bugs)

This was also discussed on mattermost: https://chat.slint.dev/public/pl/znrdbu6yujye5pximmwmye6m3c

The instructions we have are on https://github.com/slint-ui/slint/blob/master/examples/mcu-board-support/README.md#esp32-s3-box
And compiling the prinderdemo_mcu should work.

I am personally not a Windows users so I've never tried running this on Windows but it should nevertheless work the same.

**Reply by hhhdsdlfsdh** [reply]
Thank you
